Formula/m/mumps.rb

```diff
@@ -0,0 +1,55 @@
+class Mumps < Formula
+  desc "MUltifrontal Massively Parallel sparse direct Solver"
+  homepage "https://mumps-solver.org/"
+  url "https://mumps-solver.org/MUMPS_5.6.2.tar.gz"
+  sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
+  license "CECILL-C"
+
+  # Core dependencies
+  depends_on "gcc"
+  depends_on "openblas"
+  fails_with :clang
+
+  def install
+    makefile = "Makefile.G95.SEQ"
+    cp "Make.inc/" + makefile, "Makefile.inc"
+    inreplace "Makefile.inc", "-soname", "-install_name" unless OS.linux?
```

Comment:
For the version to compile under MacOS, you should include the line
` inreplace "Makefile.inc", ".so", ".dylib" unless OS.linux?`

Reply:
Ugh makefiles.

Thanks for pointing that out, good catch. It is compiling as it is now though, but is this the source of the rpath issue?

Reply:
This seems to have been it! Messed with it locally and can't replicate the CI failure anymore. 

Thanks!!

Reply:
Good luck with the test. Compiling of  MUMPS is really a hassle. Even more there is a benefit from having it in homebrew.

Reply:
Mumps compiled fine, I made a mistake installing the headers.